src/vector/backends/numpy.py

```diff
@@ -1704,14 +1734,46 @@ def __repr__(self) -> str:
 
     @property
     def azimuthal(self) -> AzimuthalNumpy:
-        """Returns the azimuthal type class for the given ``VectorNumpy4D`` object."""
-        # TODO: Add an example here - see https://github.com/scikit-hep/vector/issues/194
+        """
+        Returns the azimuthal type class for the given ``VectorNumpy4D`` object.
+
+        Example:
+            >>> import vector
+            >>> # Create a 4D vector array with (x, y, z, t) components
+            >>> vec_4d = vector.array(
+            ... [
+            ...     (1.1, 2.1, 3.1, 4.1),
+            ...     (1.2, 2.2, 3.2, 4.2),
+            ...     (1.3, 2.3, 3.3, 4.3),
+            ...     (1.4, 2.4, 3.4, 4.4),
+            ...     (1.5, 2.5, 3.5, 4.5)
+            ... ], dtype=[("x", float), ("y", float), ("z", float), ("t", float)]
+            ... )
+            >>> print(vec_4d.azimuthal)
+            [(1.1, 2.1) (1.2, 2.2) (1.3, 2.3) (1.4, 2.4) (1.5, 2.5)]
+        """
         return self.view(self._azimuthal_type)
 
     @property
     def longitudinal(self) -> LongitudinalNumpy:
-        """Returns the longitudinal type class for the given ``Vectornumpy4D`` object."""
-        # TODO: Add an example here - see https://github.com/scikit-hep/vector/issues/194
+        """
+        Returns the longitudinal type class for the given ``Vectornumpy4D`` object.
+
+        Example:
+            >>> import vector
+            >>> # Create a 4D vector array with (x, y, z, t) components
+            >>> vec_4d = vector.array(
+            ... [
+            ...     (1.1, 2.1, 3.1, 4.1),
+            ...     (1.2, 2.2, 3.2, 4.2),
+            ...     (1.3, 2.3, 3.3, 4.3),
+            ...     (1.4, 2.4, 3.4, 4.4),
+            ...     (1.5, 2.5, 3.5, 4.5)
+            ... ], dtype=[("x", float), ("y", float), ("z", float), ("t", float)]
+            ... )
+            >>> print(vec_4d.longitudinal)
+            [(3.1,) (3.2,) (3.3,) (3.4,) (3.5,)]
+        """
         return self.view(self._longitudinal_type)
 
     @property
```

Comment:
It would also be nice to have an example for the `temporal` property 🙂

Reply:
I have left it out purposely to give other first-time contributors like myself a shot at it. :)